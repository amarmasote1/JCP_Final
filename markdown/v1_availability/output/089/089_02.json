{
    "body": [
        {
            "warnings": [],
            "message": null,
            "nodeType": null,
            "body": null,
            "key": "JCP|58078781032|EACH",
            "status": 0
        }
    ],
    "message": "Successful",
    "status": 221
}