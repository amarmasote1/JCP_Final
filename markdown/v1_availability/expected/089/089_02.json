{"body":[{"status":0,"message":null,"key":"JCP|58078781032|EACH","body":null,"warnings":[]}],"status":221,"message":"Successful"}